src/main/preprocessing/code_tracker_handler.py

```diff
@@ -10,24 +10,20 @@
 from src.main.util.language_util import get_language_by_extension
 from src.main.util.strings_util import convert_camel_case_to_snake_case
 from src.main.util.consts import CODE_TRACKER_COLUMN, DEFAULT_VALUE, INVALID_FILE_FOR_PREPROCESSING, LANGUAGE, \
-    ISO_ENCODING, LOGGER_NAME
+    ISO_ENCODING, LOGGER_NAME, TEST_MODE
 
 log = logging.getLogger(LOGGER_NAME)
 
 
 def fill_column(data: pd.DataFrame, column: CODE_TRACKER_COLUMN, fits_column_restriction: Callable[[Any], bool],
                 default_value: DEFAULT_VALUE) -> Any:
     values = data[column.value].unique()
-    index = np.argwhere(np.array(list(map(default_value.is_equal, values))))
-    values = np.delete(values, index)
-    # If list is empty after removing defaults, return default value
-    if values.size == 0:
-        return default_value.value
-    # If we have only 1 valid element after removing defaults, we should return it
-    elif len(values) == 1 and fits_column_restriction(values[0]):
+    # Todo: make it more readable and delete not necessary params
+    # Delete all possible default values. If we have only 1 valid element after removing defaults, we should return it
+    # Otherwise it is INVALID_FILE_FOR_PREPROCESSING
+    values = [x for x in values if x not in [np.nan, None, np.datetime64('NaT'), 0] and not pd.isna(x)]
```

Comment:
I can be wrong, but it seems that new data can have -1 in the 'age' column, for example, because it's an initial value. So it's okay to have -1 and the actual value in the 'age' column, those files shouldn't be invalid
![image](https://user-images.githubusercontent.com/24303332/90762625-03ad3c00-e2ee-11ea-8064-63afaa532b7a.png)